### ISSUE TITLE
Configuration will not take affect

### ISSUE THREAD

**Issue by petershaw** [boxuk/jslint-jenkins-plugin#11]
Hi, 

i'm trying to get some Condifuration working, but none of it will work and affect jslint. 

I tried:

```
-Dnode=true -Dpredef=require 
```

as command in the project panel. I also tried to set a .jslintrc file in the jenkins home folder AND in the project workspace. I also put a Header into the file: /*jslint node: true, predef: require */, but none of these will take any affekt, I still got:

```
require' is not defined.  
```

Any help, how i can check node sources without this false positives? 
Thanks a lot, 
ps

**Comment by gavD** [comment]
Hi,

I'm no longer the maintainer on this product but I _think_ there's a box on the plugin configuration, either at job of Jenkins config level, that allows you to enter your JSLint options. It's a long time since I touched it though so apologies if that's not the case!
